pkg/processor/runtime/golang/runtime.go

```diff
@@ -60,6 +60,8 @@ func NewRuntime(parentLogger nuclio.Logger,
 		return nil, errors.Wrap(err, "Failed to get handler function")
 	}
 
+	newGoRuntime.MarkReady()
```

Comment:
Let's be more flexible here and make this `SetStatus` where `status` is an enum.